deploy/docker/cp-dts-status-checker/template.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <style>
+        table,
+        td {{
+            border: 1px solid black;
+            border-collapse: collapse;
+            padding: 5px;
+        }}
+    </style>
+</head>
+
+<body>
+    <p>Dear testuser,<br>
```

Comment:
it is better to have clean template as example, plese change testuser to user, as it was before. If you want to make this template differ from one that hardcoded to script, we can change another sentence. 
f.e.
`New DTS status was detected:` -> `DTS have changed their statuses:`